### PR TITLE
#619 Duplicate footer

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**

### CODE REVIEW

File: src/routes/Booksley/Booksley.module.scss [hunk 1]
```diff
@@ -374,3 +329,7 @@
   @include px(4);
   padding-top: 181px;
 }
+
+.strangeGrey {
```
Comment (CodeMeNatalie) [inline src/routes/Booksley/Booksley.module.scss]: 50 shades 😆

File: src/styles/_variables.scss [hunk 2]
```diff
@@ -376,7 +376,7 @@ $input-disabled-bg: #f0f0f0;
 
 $input-focus-color: $input-color;
 $input-focus-bg: $input-bg;
-$input-focus-border-color: #75b6ff;
+$input-focus-border-color: #2587f9;
```
Comment (CodeMeNatalie) [inline src/styles/_variables.scss]: `$input-focus-border-color: $color-blue;`

File: src/routes/Careers/Careers.js [hunk 3]
```diff
@@ -13,157 +13,156 @@ import { Rectangles } from './components/Rectangles'
...
+          </div>
+          <div className={c.photo}>
+            <Img sizes={data.hero24Image.sizes} alt="our team" />
           </div>
         </div>
       </div>
 
-      <Rectangles />
+      <div className={c.section5}>
+        <h3 className={c.title}>The perks </h3>
```
Comment (CodeMeNatalie) [inline src/routes/Careers/Careers.js]: `<h3 className={c.title}>The perks</h3>`

File: src/styles/_variables.scss (hunk 2 above)
Comment (szczepanbarszczowski) [reply src/styles/_variables.scss]: +1

File: src/routes/Careers/Careers.js (hunk 3 above)
Comment (szczepanbarszczowski) [reply src/routes/Careers/Careers.js]: +1

**Pull request merged**